lookup.js

```diff
@@ -195,6 +195,7 @@ exports.init = function (sbot, config) {
   }
 
   function branchStreamOld(rootMetafeedId) {
+    // WARNING this method needs loadState to be called?
```

Comment:
This is already handled when calling `branchStream()`. This comment can safely be removed.

Reply:
I just say your change to add defer stuff! nice!